src/style/animatableManager.js

```diff
@@ -0,0 +1,146 @@
+import _ from 'lodash';
+import * as Animatable from 'react-native-animatable';
+
+
+/** Animations Definitions */
+const definitions = {
+  indexEntrance: {
+    from: {opacity: 0, translateY: 40},
+    to: {opacity: 1, translateY: 0},
+  },
+};
+
+class AnimatableManager {
+  constructor() {
+    this.loadCustomDefinitions(definitions);
+    this.animations = getAnimations();
+  }
+
+  loadCustomDefinitions(customDefinitions) {
+    if (customDefinitions) {
+      Animatable.initializeRegistryWithDefinitions(customDefinitions);
+      this.updateDefinitions(customDefinitions);
+    }
+  }
+
+  loadSlideByHeightDefinitions(height, sufix) {
+    const definition = {};
+    // bottom
+    definition[`slideInUp_${sufix}`] = {
+      from: {translateY: height},
+      to: {translateY: 0},
+    };
+    definition[`slideOutDown_${sufix}`] = {
+      from: {translateY: 0},
+      to: {translateY: height},
+    };
+    // top
+    definition[`slideInDown_${sufix}`] = {
+      from: {translateY: -height},
+      to: {translateY: 0},
+    };
+    definition[`slideOutUp_${sufix}`] = {
+      from: {translateY: 0},
+      to: {translateY: -height},
+    };
+    // relative
+    definition[`growUp_${sufix}`] = {
+      from: {height: 0},
+      to: {height},
+    };
+    definition[`growDown_${sufix}`] = {
+      from: {height},
+      to: {height: 0},
+    };
+
+    Animatable.initializeRegistryWithDefinitions(definition);
+    this.updateDefinitions(definition);
+  }
+
+  updateDefinitions(newDefinitions) {
+    Object.assign(definitions, newDefinitions);
+    this.animations = getAnimations();
+  }
+
+  /** Presets */
+  getFadeIn(options) {
+    return {
+      animation: 'fadeIn',
+      easing: 'ease-out-quint',
+      duration: 300,
+      useNativeDriver: true,
+      ...options,
+    };
+  }
+
+  getFadeInRight(options) {
+    return {
+      animation: 'fadeInRight',
+      easing: 'ease-out-expo',
+      duration: 500,
+      useNativeDriver: true,
+      ...options,
+    };
+  }
+
+  getFadeInLeft(options) {
```

Comment:
I would rename this preset to something that explains the delay behavior and also expose the delays arrays as a param to control.

Reply:
Done.